src/libData/AccountData/AccountStoreSC.cpp

```diff
@@ -540,9 +538,11 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
       }
 
       if (is_library) {
+        // Call to library should silently passed to scilla so that the gas is
+        // charged based on the size of the message
         LOG_GENERAL(WARNING, "Library being called");
-        error_code = TxnStatus::FAIL_SCILLA_LIB;
-        return false;
+        // error_code = TxnStatus::FAIL_SCILLA_LIB;
+        // return false;
```

Comment:
```suggestion
```

Reply:
Clang-format tools takes care of it automatically.